examples/09-Incremental-Training-with-Different-Learning-Rates-and-Layer-Freezing.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>Can we add - Why do we want to use different learning rates for different layers during finetuning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>